### ISSUE TITLE
Natures compass not working with any biome 

### ISSUE THREAD

**Issue by BunE-exe** [MattCzyr/NaturesCompass#72]
I try to search for a biome, it only says

Status: Not Found - Biome: Swampland - Radius: 10001 - Samples: 24532 -

Is the mod buggy?

**Comment by MattCzyr** [comment]
As unlikely as it is, it could be the case that a swampland biome doesn't exist within a 10,000 block radius around you. Have you tried to replicate this on other biomes that you know exist nearby? For example, try searching for a forest when you know there's a forest nearby. Let me know if the issue persists.

**Comment by BunE-exe** [comment]
Ok, so from what I understood the targeted biome must pre-exist for nature's compass to locate it? And it can't locate biomes that haven't been generateded?

**Comment by MattCzyr** [comment]
No, it doesn't have to be generated already. The compass can locate biomes in chunks that haven't been generated yet. I just meant that in this case, there may not have been a swampland in that radius around you. If you retry it with a biome you know exists within that radius, it'll give me a better idea of where to start with this issue.

**Comment by ItsRohan0165** [comment]
i have the same issue , status not found
